src/components/common/Dialogs/EditDomainDialog/EditDomainDialog.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { string, object, number, boolean, InferType } from 'yup';
+import { defineMessages } from 'react-intl';
+
+import Dialog, { DialogProps, ActionDialogProps } from '~shared/Dialog';
+import { ActionHookForm as Form } from '~shared/Fields';
+
+import { Domain } from '~types';
+import { ActionTypes } from '~redux';
+import { WizardDialogType } from '~hooks';
+import { pipe, withMeta, mapPayload } from '~utils/actions';
+import { DomainColor } from '~gql';
+import { findDomainByNativeId } from '~utils/domains';
+import { notNull } from '~utils/arrays';
+import { getDomainOptions } from '~shared/DomainFundSelectorSection/helpers';
+
+import EditDomainDialogForm from './EditDomainDialogForm';
+import { getEditDomainDialogPayload, notRootDomain } from './helpers';
```

Comment:
As above

Reply:
Do you mean it's appearing as if they are unused? It's not the case for me :thinking: